lib/ansible/plugins/callback/unixy.py

```diff
@@ -18,6 +18,15 @@
       - default_callback
     requirements:
       - set as stdout in configuration
+    options:
+      show_only_changed:
+        description: If true, do not show output from 'ok' task results
+        default: False
+        env:
+          - name: UNIXY_SHOW_ONLY_CHANGED
```

Comment:
If you're defining a new environment variable, it should be prefixed with`ANSIBLE_`

Reply:
I pulled this out, but will keep in mind for next time. Thanks for the feedback.